css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,129 @@
+$sc-padding: 0.5em 1em !default;
+$sc-padding-md: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-5;
+	grid-template-columns: max-content auto max-content;
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	&.segmented-lg {
```

Comment:
don't forget to add `-sm` here as well

Reply:
@ollips  - Taking a look at this, the scale works well for making it LG.  The sm version might be superfluous as I understand it.

Reply:
Hmm I don't remember exactly what I was referring too 😅 Probably for consistency with other elements, usually if we have large size we also expect the small variation

Reply:
This should be updated now.